### ISSUE TITLE
Wrong week number. Today (3 January 2017) it emits week 0.

### ISSUE THREAD

**Issue by unagitated** [datetime/week#3]
Wrong week number. Today (3 January 2017) it emits week 0.

**Comment by sindrenm** [comment]
Yeah, and this week is week 11, not 10 … :neutral_face:

**Comment by YounGoat** [comment]
The starting day of _week 1_ is sometimes confusing. According to MySQL function `WEEK()`, there are 7 different modes to define the start day of _week 1_ and the number of week before _week 1_.  
See package [dafo.getWeekOfYear()](https://www.npmjs.com/package/dafo#others) for a smiliar implementation in Node.js .

**Comment by sindrenm** [comment]
ISO 8601 defines the first week as the week with the first Thursday in it. The first Thursday in 2018 would be January 4. That makes the current week (the one with January 25 in it) the 4th week of the year.

This package reports today as being in week 3. If the package does not intend to follow ISO 8601 to determine which week it is, then I suggest that at least be documented in the README to avoid further confusion.

**Comment by YounGoat** [comment]
@sindrenm You are right!  
According to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), the first day of week is Monday, and week 1 is the first week has 4 or more days in this year (this description is equivalent to *the week with the first Thursday in it*), so mode 3 used in MySQL `WEEK()` is the right choice( see [Week Mode](https://www.npmjs.com/package/dafo#week-mode) for details):
```javascript
const dafo = require("dafo");
dafo.getWeekOfYear(new Date(2018, 0, 25), /* mode */ 3);
// RETURN 4
```
And with mode 0 or 2, it returns 3.

**Comment by sindrenm** [comment]
Yep, `dafo` seems to do the trick when passed mode 3, but this package (_week_) still doesn't. However, I'm not sure if it's maintained. @jonschlinkert? :smile:

**Comment by jonschlinkert** [comment]
> However, I'm not sure if it's maintained. @jonschlinkert?

This is a Free and Open Source software project, maintained by myself and other devs in their free time. Don't make comments that are explicitly intended to make someone feel guilty and obligated to respond to you in an expedient way. 

I usually block people for making divisive or manipulative comments, but since this project does not have a contributing guide I'll give you a warning.

**Comment by sindrenm** [comment]
Sorry, @jonschlinkert, it was not meant in such a way at all. It was meant as a friendly question, but I understand how it might have come out as a bit snarky. I am grateful for the project and contribution to OSS. After all, I was using your package, so I should be. :smile: 

So I don't want you to feel guilty, if anything I want you to be proud of the fact that you made this package available for everyone to use. :+1: 

Now that that's in the clear, I think it would be good to have contributing guidelines in general, but it's not a necessity. And if pull requests are welcome, then that's fantastic. :smile: 

Are we good?
